### PR TITLE
add recipe for r-txtq

### PR DESCRIPTION
**Pull request by brendanf**
- [X] License file is packaged
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
  [see here](https://github.com/wlandau/txtq/issues/14#issue-479171609)

### CODE REVIEW

File: recipes/r-txtq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+about:
+  home: https://github.com/wlandau/txtq
+  license: MIT
+  summary: This queue is a data structure that lets parallel processes send and receive messages,
+    and it can help coordinate the work of complicated parallel tasks. Processes can
+    push new messages to the queue, pop old messages, and obtain a log of all the messages
+    ever pushed. File locking preserves the integrity of the data even when multiple
+    processes access the queue simultaneously.
+  license_family: MIT
+  license_file: LICENSE.md
```
Comment (dbast) [inline recipes/r-txtq/meta.yaml]: Please change this to:
```
license_file:
  - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
  - LICENSE
``` 

If you update your installed conda-build version, this should have been done this way already.

**Comment by dbast** [comment]
Please read the comments instead of adding the LICENSE.md file.

File: recipes/r-txtq/meta.yaml (hunk 1 above)
Comment (brendanf) [reply recipes/r-txtq/meta.yaml]: Sorry, didn't see this before making the other change.  Thanks for the help!

**Comment by brendanf** [comment]
@conda-forge/help-r This should be ready to go now.

**Pull request merged**